public/app/plugins/datasource/prometheus/language_provider.ts

```diff
@@ -95,12 +105,13 @@ export default class PromQlLanguageProvider extends LanguageProvider {
       console.error(error);
     }
 
-    return [];
+    return defaultValue;
   };
 
   start = async (): Promise<any[]> => {
-    this.metrics = await this.request('/api/v1/label/__name__/values');
+    this.metrics = await this.request('/api/v1/label/__name__/values', []);
     this.lookupsDisabled = this.metrics.length > this.lookupMetricsThreshold;
+    this.metricsMetadata = await this.request('/api/v1/metadata', {});
```

Comment:
Could make this conditional on `lookupsDisabled`, but the responses seems small and fast enough.